h/auth/util.py

```diff
@@ -157,25 +149,42 @@ def check_auth_client(username, password, request):
     if not hmac.compare_digest(client.secret, client_secret):
         return None
 
-    return principals_for_auth_client(client)
+    return client
 
 
 def principals_for_auth_client(client):
     """
     Return the list of additional principals for an auth client
 
-    :type client: h.models.AuthClient
+    :type client: :py:class:`h.models.auth_client.AuthClient`
     :rtype: list
     """
 
     principals = set([])
 
-    principals.add('auth_client:{authority}'.format(authority=client.authority))
+    principals.add('client:{client_id}@{authority}'.format(client_id=client.id, authority=client.authority))
```

Comment:
Why the change from `auth_client:` to `client:`?

Reply:
`auth_client` felt H-jargon-y to me; I can change it back if you like!